routes/credentialsRoutes.js

```diff
@@ -0,0 +1,61 @@
+const router = require("express").Router();
+const { ContentCreatorApplication } = require("../models/ContentCreatorApplication");
+const bcrypt = require('bcrypt');
+
+const jwt = require("jsonwebtoken");
+const { compare, encrypt } = require("../helpers/password");
+
+// Content Creator Application Route
+router.post("/signup", async (req, res) => {
```

Comment:
Above this function should be a doxygen comment. If you haven't used those before I'll give a quick summation, but you should read up on it a bit as well. It's a great way to make function comments or comments on larger bits of code, (and it's one of the guidelines we put into definition of done😉).

They're really easy to make with co-pilot, or with "Doxygen Documentation Generator" extension (though that one can be a bit finicky when the function is set up like this and not like a classic function).

The comment would look something like this for this code:
/**
 \* Signup Route.
 \* Creates a new Content Creator Application.
 \* 
 \* @param {JSON} Form Which includes the following fields:
 \*  @param {String} name Name of the Content Creator
 \*  @param {String} email Email of the Content Creator (Will be used for login)
 \*  @param {String} password Password of the Content Creator (Will be encrypted)
 \* @returns {JSON} Returns the created Content Creator Application
 */

You basically type in "/**" press enter and it does the rest.


/**
\* "Description"
\* 
\* "params" if applicable
\* "return value" if applicable
*/ 

But there is more to it, so take a skim through the wiki page😅